### ISSUE TITLE
Do I need to do special processing on the co3d dataset in order to train idr?

### ISSUE THREAD

**Issue by CoolChameleon** [facebookresearch/pytorch3d#1145]
Hi! I'm trying to reproduce some of your experiments on co3d dataset, but when I try to train a single sequence idr model, I find the loss of the model on the training set just does not decrease since the 2nd epoch, and after 400 epochs of training, I still get a sphere like this

https://user-images.githubusercontent.com/23183849/160677359-aec41bdd-1a6a-48d5-bb51-645fb3604e60.mp4

I remembered that in the IDR paper, the authors mentioned that they normalize the DTU dataset to get the objects to lie near the origin of the world coordinate system, does the co3d dataset need similar processing?

**Comment by gkioxari** [comment]
In general, various 3D methods out there, including IDR, expect data to be in a specific form (e.g. centered, normalized, in a specific coordinate system etc.). It's likely that you need to massage the data you are working with (in this case CO3D, but it holds for any data) to IDR's specifications in order to make the IDR training loop work as expected. 

The team is thinking about adding IDR support for CO3D, but that is not available yet.

**Comment by gkioxari** [comment]
Can you provide the full training script and the data on which you are training, so that we can re-produce the error?

**Comment by CoolChameleon** [comment]
I used the following command to run my experiment:
`./experiment.py --config-name repro_singleseq_idr dataset_args.dataset_root=/media/disk2/co3d dataset_args.category='toybus'  exp_dir=./data/exp/0329`

**Comment by CoolChameleon** [comment]
Hello! I'm running more experiments using your code, and I find when I use nerf, the code runs just fine. However, when I tried to use IDR to reproduce some results in CO3D paper, it will produce similar situation as above. If you have used similar normalizing operations in your experiments, can you share the normalization script you used? Thanks!

**Comment by davnov134** [comment]
Hi, thanks for the questions about Impliciton.
The CO3D data should be normalized already when you load it from the dataloader so no need to do additional preprocessing.
We will run your experiment and report whether we can reproduce your results.

**Comment by bottler** [comment]
I think we have observed differences in how well the IDR code trains on different GPUs, which are not yet understood.
